src/Server/types.tsx

```diff
@@ -0,0 +1,40 @@
+export type fetchListProps = {
+  lat: number;
+  lng: number;
+  dist: number;
+  rad: number;
+  type: string;
```

Comment:
As another dev I wouldn't know what those variables mean. If there is a chance to rename them I would consider it.